app/test_main.py

```diff
@@ -1 +1,43 @@
-# write your code here
+import pytest
+
+from unittest import mock
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "valid_url,internet_connection,expected",
+    [
+        pytest.param(
+            True,
+            True,
+            "Accessible",
+            id="URL is valid, has internet connection"),
```

Comment:
```suggestion
            id="URL is valid, has internet connection"
        ),
```
It’s picky but could you provide this code style all over the test data